### ISSUE TITLE
Indent do blocks in Haddock?

### ISSUE THREAD

**Issue by tomjaguarpaw** [bitemyapp/esqueleto#202]
Shouldn't the do blocks, here and elsewhere, be indented?

https://github.com/bitemyapp/esqueleto/blob/4dbd5339adf99e1f045c0a02211a03c79032f9cf/src/Database/Esqueleto/Experimental.hs#L159-L162

**Comment by charukiewicz** [comment]
Hi, I wrote the `Database.Esqueleto.Experimental` module docs after @belevy finished the writing the module. Others may have a different opinion, but the short answer to your question is no, unless your personal preferences dictate otherwise.

In the case of the highlighted query, which is very short, we decided it would be fine to leave it unindented because the equivalent SQL would also probably not be intended. E.g.

```sql
SELECT *
FROM person
WHERE person.name = 'John'
```

If you don't agree and think that indentation here would help with readability, then in your code you should indent queries like this. For the later examples, which are larger and in some cases include subqueries and unions, we decided that indentation made sense because it did help with readability.

In short, just like when writing SQL, there's more than one way to write and indent this Esqueleto syntax. Ultimately it's a matter of personal preference. I'm sure if you were to install some of the highly opinionated Haskell code formatters, they would indent this code differently than we did. We use `stylish-haskell`, which only applies a minimal amount of formatting, so a lot of the code we write ends up being our personal style.

**Comment by tomjaguarpaw** [comment]
Interesting. I see this style of indentation so rarely that I assumed it must be invalid syntax! Turns out it's perfectly valid. Therefore I'm closing this issue.
